tests/onnx/test_onnx_ops_impl.py

```diff
@@ -52,9 +52,10 @@ def test_numpy_gemm(alpha, beta, trans_a, size_a, trans_b, size_b):
 
     # Can be a bit different if we have floats
     if isinstance(alpha, float) or isinstance(beta, float):
-        assert numpy.isclose(
+
+        assert numpy.allclose(
```

Comment:
you can use your new `check_float_array_equal` here as well !

Reply:
got and expected are not arrays, that's why we can't use check_float_array_equal